spec/features/users/sign_in_spec.rb

```diff
@@ -981,13 +981,27 @@
           allow(IdentityConfig.store).to receive(:compromised_password_randomizer_threshold).
             and_return(2)
         end
-        it 'should bring user to compromised password page' do
+        it 'should bring user to manage password page with warning' do
           visit new_user_session_path
           fill_in_credentials_and_submit(user.email, user.password)
           fill_in_code_with_last_phone_otp
           click_submit_default
 
-          expect(current_path).to eq user_password_compromised_path
+          expect(current_path).to eq manage_password_path
+        end
+
+        it 'should redirect user to after_sign_in_path after editing password' do
+          visit new_user_session_path
```

Comment:
Could we simulate starting from the SP instead, and making sure they're eventually redirected back? I think it'd be important to make sure that's working correctly, especially when reusing the edit password code that would normally take someone back to the account page (not what we're expecting when signing in to an SP).